src/Directory.Build.props

```diff
@@ -3,9 +3,9 @@
 
   <PropertyGroup>
     <Copyright>2015-2024 The Neo Project</Copyright>
-    <VersionPrefix>3.7.4</VersionPrefix>
```

Comment:
This is no longer needed?

Reply:
`github` action step adds it; a-way of making sure you `override` or forgot to change.
```
-p:VersionPrefix=${{ env.APP_VERSION }} # Set the version prefix from tag_name
```